ai/STA/Tactic/tactic.py

```diff
@@ -11,22 +12,10 @@
 __author__ = 'RobocupULaval'
 
 
-class Tactic(object):
-    """
-        Classe mère de toutes les tactiques
-    """
+class Tactic:
 
-    def __init__(self, game_state: GameState, player: Player, target: Pose=Pose(), args: List=None):
-        """
-        Initialise la tactic avec des valeurs
-
-        :param game_state: L'état du monde pour le jeu en cours
-        :param player: Le joueur executant la tactic
-        :param target: Pose général pouvant être utilisé par les classes enfants comme elles veulent
-        """
-        assert isinstance(game_state, GameState), "Le game_state doit être un GameState"
-        assert isinstance(player, Player), "Le player doit être un Player, non un '{}'".format(player)
-        assert isinstance(target, Pose), "La target devrait être une Pose"
```

Comment:
Could you keep the target is pose check? It saved me hours quite a few times.

Reply:
yup